main/res/values/changelog_master.xml

```diff
@@ -2,5 +2,8 @@
 <resources>
   <!-- changelog for the master branch -->
   <string name="changelog_master" translatable="false">
+    <b>Next feature release:</b>\n
+    · [nothing yet]\n
```

Comment:
not exactly true, there are already some changes in master

Reply:
I will add those later, but also encourage all contributors to add their contributions to the changelog (if relevant)